pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -500,6 +516,28 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 					Field:   field.Child("domain", "devices", "networks").Index(0).Child("pod").String(),
 				})
 			}
+
+			if iface.BootOrder != nil {
+				order := *iface.BootOrder
+				// Verify boot order is greater than 0, if provided
+				if order < 1 {
```

Comment:
Perhaps a dumb question, but isn't boot order defined as an unsigned int? Why not just allow `0` as a valid order index and not worry about it?

Reply:
order "0" is not valid in libvirt.